tests/manual_download_data.html

```diff
@@ -184,12 +184,12 @@
         return Module['downloadedData'];
       }
 
-      var dataDownload = download('manual_download_data.data').then(function(data) {
+      var dataDownload = download('/test/manual_download_data.data').then(function(data) {
         console.log('downloaded data file');
-        Module.manuallyDownloadedData = 1;
```

Comment:
Why remove this?

Reply:
This check is redundant - here we check if `getPreloadedPackage` method in JS was called by checking for `Module.manuallyDownloadedData` ; however, we are already checking for the file existence and it's contents, which can only succeed if `getPreloadedPackage` was called (since .data is not available any more in server root)

Reply:
What if `getPreloadedPackage` didn't exist or didn't run though (e.g. if it was renamed)?  Wouldn't the file_packager code still run to load the preloaded data in the normal way that `--preload-file` does?   .. and therefore the file would exist even if `getPreloadedPackage` wasn't called?

Reply:
> Wouldn't the file_packager code still run to load the preloaded data in the normal way that --preload-file does?

Yes. But it would fail to load the file the normal way (which is to try and find the .data file at the root of the server)